docs/app/Examples/elements/Segment/Groups/SegmentNestedSegmentsExample.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+import {Segment, Segments} from 'stardust';
+
+export default class SegmentNestedSegmentsExample extends Component {
+  render() {
+    return (
+      <Segments>
+        <Segment>Top</Segment>
+        <Segments>
+          <Segment>Nested Top</Segment>
+          <Segment>Nested Middle</Segment>
+          <Segment>Nested Bottom</Segment>
+        </Segments>
+        <Segments className='horizontal'>
+          <Segment>Top</Segment>
+          <Segment>Middle</Segment>
+          <Segment>Bottom</Segment>
+        </Segments>
+        <Segment>Bottom</Segment>
+      </Segments>
+    );
+  }
+}
```

Comment:
Posing a question here - would it make sense here for the text to be taken in as a child and have all these be self closing? As in - `<Segment caption="Nested Top" />`. From a readability standpoint, it feels a bit cluttered.